src/components/Collapse.js

```diff
@@ -68,21 +68,11 @@ Collapse.propTypes = {
    */
   key: PropTypes.string,
 
-  /**
-   * HTML tag to use for the collapse contents. Default: div.
-   */
-  tag: PropTypes.string,
-
   /**
    * Whether collapse is currently open.
    */
   is_open: PropTypes.bool,
 
-  /**
-   * Set to True when using a collapse inside a navbar.
-   */
-  navbar: PropTypes.bool,
-
```

Comment:
Rather than removing this and creating the new `NavbarCollapse` component, could we move the logic for that component into this file?

I think it might be possible to do something like

```jsx
const Component = navbar ? Navbar.Collapse : Collapse;

return <Component in={is_open} {...otherProps} />
```

then we can somewhat reduce the number of changes in the api. What do you think?

Reply:
good idea - I'll do that and remove the NavbarCollapse component